src/Features/Core/Portable/PopulateSwitch/PopulateSwitchExpressionHelpers.cs

```diff
@@ -20,6 +20,13 @@ public static ICollection<ISymbol> GetMissingEnumMembers(ISwitchExpressionOperat
             var switchExpressionType = switchExpression?.Type;
 
             var enumMembers = new Dictionary<long, ISymbol>();
+
+            // Check if the type of the expression is a nullable INamedTypeSymbol
+            // if the type is both nullable and an INamedTypeSymbol extract the type argument from the nullable
+            // and check if it is of enum type
+            if (switchExpressionType != null)
```

Comment:
null check not needed.  `IsNullable` is null-safe (and we try to make all our `.Is` extensions that way).